DOC-3329: Premium Plugins Installation and Bundling Documentation for NPM (#3928)
tinymce/tinymce-docs: modules/ROOT/partials/configuration/skin_url.adoc

[[skin_url]]
== `+skin_url+`

This option specifies the location of the skin directory. This is useful when loading {productname} and the skin files from different locations.

When using `+skin_url+`, the `+skin+` option is not required. The `+skin_url+` option points directly to the skin directory location.

*Type:* `+String+`

=== Example: using `+skin_url+`

[source,js]
----
tinymce.init({
  selector: 'textarea',  // change this value according to the HTML
  skin_url: '/css/mytinymceskin' // change this to point to the folder of skin files
});
----

=== Example: loading premium skins from `+tinymce-premium+` (NPM)

> When installing the `+tinymce-premium+` NPM package, premium skins are stored in `+node_modules/tinymce-premium/skins/ui/+`. Configure the `+skin_url+` to point at the desired skin directory. For example:

Reviewer: ```suggestion
Configure the `+skin_url+` to point at the appropriate location. For example:
```

[source,js]
----
tinymce.init({
  selector: 'textarea',

>   skin_url: '/node_modules/tinymce-premium/skins/ui/fabric'

Reviewer: ```suggestion
  skin_url: '/<path_to_premium_skins>/ui/fabric'
```

});
----

[NOTE]
====

> * Ensure the `+node_modules+` directory (or a copy of the skin directory) is accessible to the browser. Many projects copy the required skin into a public folder (such as `+/public/skins/ui/oxide+`) during their build step and update `+skin_url+` accordingly.

Reviewer: ```suggestion
* Ensure the skin directory is accessible to the application. Many projects copy the required skin into a static assets folder (such as `+/public/skins/ui/oxide+`) during their build step.
```
